src/infra/dataverse/sparql/sparqlGateway.ts

```diff
@@ -39,6 +41,7 @@ export const sparqlGateway: DataversePort = {
       PREFIX datasetMetadata: <https://ontology.okp4.space/metadata/dataset/>
       PREFIX zoneMetadata: <https://ontology.okp4.space/metadata/zone/>
       PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+      PREFIX svccat: <https://ontology.okp4.space/thesaurus/service-category/> 
```

Comment:
To maintain consistency, I would recommend to standardize the style of prefixes within `RDF` ontologies. In your code, camel case style was used, while here lower case style is being utilized. Moreover, although prefixes can be named arbitrarily, it is generally a good thing to adopt standardized prefixes to ensure clarity and recognition (e.g. `foaf`, `rdfs`, `cc`).

I think we should rename the other prefixes to camel case for all prefixes.

Reply:
Yes, of course 😌
I'll suggest to follow the `lowercase` convention used on the ontology classes.